module1/homework/grayscaleImages/test.cpp

```diff
@@ -1,16 +1,26 @@
-#include <algorithm>
 #include <array>
 #include <utility>  // for std::pair<>
 #include <vector>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
Array and vector are included in header,so you could remove them here.

Reply:
+1